webapp-django/crashstats/cron/management/commands/cronrun.py

```diff
@@ -117,8 +117,6 @@ def _run_one(self, job_spec, force=False, cmd_args=None):
         logger.info("about to run %s", cmd)
 
         now = timezone.now()
-        log_run = True
-        exc_type = exc_value = exc_tb = None
```

Comment:
We don't need any of these since we'll now call `self._log_run` after every successful run_time and in the event of failure. We no longer call it in the `finally` block after everything has happened.